pkg/generator/steps/layout/tpl-new/py-services/_service_/app/service_extra.py.tpl

```diff
@@ -0,0 +1,4 @@
+class ServiceExtra:
+    def __init__(self, service_context):
```

Comment:
Can we use python type hints here? It will be much better if the user could see what is service_context

Reply:
And in other places, where we export our code to the user. It will give him better IntelliSense.

Reply:
Not in this place in particular, because it will cause circular import error, in other places I tried to use types, but the code need another round of adding types later